pkg/minishift/constants/constants.go

```diff
@@ -30,3 +35,13 @@ const (
 var (
 	ValidIsoAliases = []string{B2dIsoAlias, CentOsIsoAlias, MinikubeIsoAlias}
 )
+
+// ProfileAuthorizedKeysPath returns the path of authorized_keys file in profile dir used for authentication purpose
+func ProfileAuthorizedKeysPath() string {
+	return filepath.Join(constants.Minipath, "authorized_keys")
```

Comment:
Even function name can be changed to ` ProfilePublicKeyPath()`

Reply:
I think this is the `authorized_keys` file which contains the public keys related to users that are authorized to ssh into the server. This can be put inside the certs dir... but the file name should not be changed.

Reply:
> Even function name can be changed to ProfilePublicKeyPath()

Function name can't be changed as it will confuse the reader of code since ssh authentication include private and authorized_keys file.

> return filepath.Join(constants.Minipath, "certs" ,"id_rsa.pub")

Sure.